Model/GridSourceType/RepositorySourceType/HyvaAdminEventContainer.php

```diff
@@ -0,0 +1,23 @@
+<?php declare(strict_types=1);
+
+namespace Hyva\Admin\Model\GridSourceType\RepositorySourceType;
+
+class HyvaAdminEventContainer
+{
+    private $hyvaAdminContainer;
```

Comment:
I find it a bit irritating that a `HyvaAdminEventContainer` receives a `$hyvaAdminContainer` as a argument.
How can we make the argument name more descriptive?
Maybe `$hyvaAdminEventPayload` or `$hyvaAdminEventObject`? Any idea?

Reply:
I've checked whether there is a name more related to events, but didn't find anything that convinced me. I think $hyvaAdminEventObject should is ok :)